hkmc2/shared/src/main/scala/hkmc2/semantics/Elaborator.scala

```diff
@@ -826,9 +891,11 @@ extends Importer:
               // * Elaborate signature
               val st = td.annotatedResultType.orElse(newSignatureTrees.get(id.name))
               val s = st.map(term(_)(using newCtx))
-              val b = rhs.map(term(_)(using newCtx))
+              val b = if ctx.mode != Mode.Light
```

Comment:
It looks like this is the only place `ctx.mode` is used. Are you sure there are no other places where it'd be appropriate?

Reply:
I checked again and AFAIK that's the only place where we don't elaborate if the mode is light. Given that the TermDef body terms are the only terms that we want to omit.